src/sentry/static/sentry/app/views/settings/incidentRules/triggers/chart/thresholdsChart.tsx

```diff
@@ -263,16 +264,27 @@ export default class ThresholdsChart extends React.PureComponent<Props, State> {
       selected,
     };
 
+    const chartOptions = {
+      tooltip: {
+        valueFormatter: (value, seriesName) => {
+          return tooltipFormatter(value, seriesName);
+        },
+      },
+      yAxis: {
+        max: this.state.yAxisMax ?? undefined,
+        axisLabel: {
+          formatter: (value: number) => axisLabelFormatter(value, data[0].seriesName),
```

Comment:
Probably want to check for `data.length` here, sometimes `data` is empty don't want to throw an error